pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -75,6 +75,10 @@ func validateNoConflict(presentLabels []string, absentLabels []string) error {
 // with an additional check for .labelSelector to be nil.
 func ValidatePodTopologySpreadArgs(args *config.PodTopologySpreadArgs) error {
 	var allErrs field.ErrorList
+	if args.DisableDefaultConstraints && len(args.DefaultConstraints) > 0 {
+		p := field.NewPath("disableDefaultConstraints")
+		allErrs = append(allErrs, field.Invalid(p, args.DisableDefaultConstraints, "can not be true when there .defaultConstraints are not empty"))
+	}
```

Comment:
not sure whether we should return here, to avoid return too much error info to confuse the user.

Reply:
We should gather all errs and then return